perl-lib/OESS/lib/OESS/MPLS/FWDCTL.pm

```diff
@@ -521,7 +521,14 @@ sub _register_rpc_methods{
                                   required => 1,
                                   pattern => $GRNOC::WebService::Regex::INTEGER);
     $d->register_method($method);
-
+    $method = GRNOC::RabbitMQ::Method->new( name        => 'is_online',
+                                            async       => 1,
+                                            callback    => sub { my $method = shift;
+                                                                 $self->{'logger'}->error('Inside is_online FWDCTL');
```

Comment:
This should be set to debug if logged at all.

Reply:
removed